src/main/java/net/dv8tion/jda/api/entities/GuildChannel.java

```diff
@@ -179,6 +179,15 @@
     @Nonnull
     List<PermissionOverride> getRolePermissionOverrides();
 
+    /**
+     * Returns whether or not this GuildChannel is synced.
+     * <br>A GuildChannel is synced when its {@link net.dv8tion.jda.api.entities.PermissionOverride PermissionOverrides} match
+     * those of {@link #getParent() its parent category}. If the GuildChannel doesn't have a parent category will this return true.
```

Comment:
```suggestion
     * those of {@link #getParent() its parent category}. If the GuildChannel doesn't have a parent category this will return true.
```